lxml/cssselect.pyi

```diff
@@ -0,0 +1,15 @@
+from typing import Dict, List, Optional, Union
+
+from lxml import  etree
+
+
+# _AnyStr = Union[str, bytes]
+# _ListAnyStr = Union[List[str], List[bytes]]
+_DictAnyStr = Union[Dict[str, str], Dict[bytes, bytes]]
```

Comment:
Can this be a Mapping instead?

Reply:
i copied this from the `etree` stub.

i'm not sure whether the `etree._NSMap` type would be appropriate here. for the `xpath` method, which i think is used eventually, it wouldn't as this one doesn't accept a mapping with a default namespace (`None` as key`).